database/factories/Multiplayer/EventFactory.php

```diff
@@ -37,3 +37,9 @@
         'text' => 'PART',
     ];
 });
+
+$factory->state(App\Models\Multiplayer\Event::class, 'game', function (Faker\Generator $faker) {
+    return [
+        'text' => 'test game',
```

Comment:
Should probably also create/associate a game with this event for it to be valid. Also `user_id` should be `null` for this event type.